ui-core/src/components/inputs/checkbox/CheckboxesTree.tsx

```diff
@@ -0,0 +1,97 @@
+import { useState, useCallback } from 'react';
+import CheckboxList from './CheckboxList';
+import { buildRelationshipMaps, updateItemStatesOptimized } from './updateItemState';
+import React from 'react';
+import FieldWrapper from '../FieldWrapper';
```

Comment:
Should we keep osrd guidelines for osrd-ui as well and use absolute path for case like this ?

Could we also keep the eslint config to sort imports and for types ?

Reply:
I create a new issue to handle this subject -> https://github.com/OpenRailAssociation/osrd-ui/issues/71